### PR TITLE
Resolves #2766

### LINKED ISSUE #2766: Format-Volume -WhatIf parameter is ignored
I was exploring the Format-Volume cmdlet using the -WhatIf parameter. It was ignored and formatted my drive without any questions asked. I've now done it two times just to confirm.

I posted about this at https://docs.microsoft.com/en-us/answers/questions/622675/powershell-whatif-parameter-was-ignored.html and got referred to post a feedback on the Format-Volume documentation page.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 96c41cc9-a100-86fb-b6ac-64521347c761
* Version Independent ID: dc541d59-bbc3-cc3d-9b98-79d06870564f
* Content: [Format-Volume (Storage)](https://docs.microsoft.com/en-us/powershell/module/storage/format-volume?view=windowsserver2019-ps)
* Content Source: [docset/winserver2019-ps/storage/Format-Volume.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2019-ps/storage/Format-Volume.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend*
powershell_2021-11-10_20-49-04
powershell_2021-11-10_21-30-09

*

### PR DESCRIPTION
**Pull request by velkovb**
Add note that whatif is not supported.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2766

### CODE REVIEW

File: docset/winserver2012-ps/storage/Format-Volume.md [hunk 1]
```diff
@@ -388,6 +388,8 @@ Accept wildcard characters: False
 Shows what would happen if the cmdlet runs.
 The cmdlet is not run.
 
+NOTE: The WhatIf switch does not work with this cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/storage/Format-Volume.md:391]:
```suggestion
**NOTE**: The WhatIf switch does not work with this cmdlet.
```

File: docset/winserver2012r2-ps/storage/Format-Volume.md [hunk 2]
```diff
@@ -408,6 +408,8 @@ Accept wildcard characters: False
 Shows what would happen if the cmdlet runs.
 The cmdlet is not run.
 
+NOTE: The WhatIf switch does not work with this cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/storage/Format-Volume.md:411]:
```suggestion
**NOTE**: The WhatIf switch does not work with this cmdlet.
```

File: docset/winserver2016-ps/storage/Format-Volume.md [hunk 3]
```diff
@@ -448,6 +448,8 @@ Accept wildcard characters: False
 Shows what would happen if the cmdlet runs.
 The cmdlet is not run.
 
+NOTE: The WhatIf switch does not work with this cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/storage/Format-Volume.md:451]:
```suggestion
**NOTE**: The WhatIf switch does not work with this cmdlet.
```

File: docset/winserver2019-ps/storage/Format-Volume.md [hunk 4]
```diff
@@ -448,6 +448,8 @@ Accept wildcard characters: False
 Shows what would happen if the cmdlet runs.
 The cmdlet is not run.
 
+NOTE: The WhatIf switch does not work with this cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/storage/Format-Volume.md:451]:
```suggestion
**NOTE**: The WhatIf switch does not work with this cmdlet.
```

File: docset/winserver2022-ps/storage/Format-Volume.md [hunk 5]
```diff
@@ -448,6 +448,8 @@ Accept wildcard characters: False
 Shows what would happen if the cmdlet runs.
 The cmdlet is not run.
 
+NOTE: The WhatIf switch does not work with this cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/storage/Format-Volume.md:451]:
```suggestion
**NOTE**: The WhatIf switch does not work with this cmdlet.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by dariomws** [comment]
@velkovb, could you test it?

**Comment by velkovb** [comment]
@dariomws Yes, I have tested it. I have memories that I have done it before as well but it might be a different cmdlet from this module.

**Review by dariomws — APPROVED** [review]

**Comment by dariomws** [comment]
@velkovb thanks!

**Review by tseward — APPROVED** [review]

**Pull request merged**